### PR TITLE
integrate gamesdk source code

### PR DESCRIPTION
**Pull request by bofeng-song**

### CODE REVIEW

File: sources/CMakeLists.txt [hunk 1]
```diff
@@ -16,6 +16,13 @@ set(CC_TINYDIR_SOURCES
   ${CMAKE_CURRENT_LIST_DIR}/tinydir/tinydir.h
 )
 
+set(CC_GAME_ACTIVITY_SOURCES
```
Comment (PatriceJiang) [inline sources/CMakeLists.txt]: Only on Android devices ?

Comment (minggo) [reply sources/CMakeLists.txt]: Yep, it is better to set it only on Android.

**Comment by minggo** [comment]
Please add the source code in 3rd party library.

File: sources/CMakeLists.txt (hunk 1 above)
Comment (bofeng-song) [reply sources/CMakeLists.txt]: pr has been fixed

**Review by PatriceJiang — APPROVED** [review]

**Review by minggo — APPROVED** [review]

**Pull request merged**